pyrdp/layer/layer.py

```diff
@@ -1,6 +1,6 @@
 #
 # This file is part of the PyRDP project.
-# Copyright (C) 2018 GoSecure Inc.
+# Copyright (C) 2018, 2019, 2021 GoSecure Inc.
```

Comment:
Can't we just write 2018-2021? Here and a couple other places.

Reply:
(basically everywhere we have more than 1 year)

Reply:
Unfortunately no, where I wrote without dashes are places where there were no code changes in the missing years. I'm not sure if we can apply the copyright statement if there was no change.

Reply:
ah ok then